### ISSUE TITLE
Deprectation warning returnOriginal mongo

### ISSUE THREAD

**Issue by Moumouls** [parse-community/parse-server#7623]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description
Getting

```
collection.findOneAndUpdate option [returnOriginal] is deprecated and will be removed in a later version.
```
### Steps to reproduce
Pull master branch and run trigger findOneAndUpdate on db adapter

### Actual Outcome
A lot of deprecation warnings

### Expected Outcome
No warning
### Environment
`5.0.0-alpha.1`

We need to replace `returnOriginal` with `returnDocument` as suggested here: https://jira.mongodb.org/browse/NODE-1812

**Comment by Moumouls** [comment]
@mtrezza  related: https://github.com/parse-community/parse-server/issues/7624#issuecomment-942422705

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-alpha.2](https://github.com/parse-community/parse-server/releases/tag/5.0.0-alpha.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/parse-server/releases/tag/5.0.0)
